SPECS/mariadb/mariadb.spec

```diff
@@ -23,11 +23,13 @@ BuildRequires:  pcre2-devel
 BuildRequires:  pkgconf
 BuildRequires:  systemd-devel
 BuildRequires:  zlib-devel
+Requires:       %{name}-connector-c
+Requires(postun): shadow-utils
+Requires(pre):  shadow-utils
```

Comment:
I think since the individual pre/postun sections only apply to the server subpackage, these requirements should go under that subpackage?

Reply:
Good point. Will update